### PR TITLE
Edits on Documentation README.md

### PR DESCRIPTION
**Pull request by DougTW**
TM&B pass. Basic editorial changes. Added some descriptions to bullet items.

Signed-off-by: Doug Martin <doug.martin@intel.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,39 +20,39 @@ For details of the other Kata Containers repositories, see the
...
 
-* [GPU Passthrough with Kata](./use-cases/GPU-passthrough-and-Kata.md)
-* [OpenStack Zun with Kata Containers](./use-cases/zun_kata.md)
-* [SR-IOV with Kata](./use-cases/using-SRIOV-and-kata.md)
-* [Intel QAT with Kata](./use-cases/using-Intel-QAT-and-kata.md)
-* [VPP with Kata](./use-cases/using-vpp-and-kata.md)
-* [SPDK vhost-user with Kata](./use-cases/using-SPDK-vhostuser-and-kata.md)
+* [GPU Passthrough with Kata Containers](./use-cases/GPU-passthrough-and-Kata.md): Kata Containers supports passing certain GPUs from the host into the container.
+* [OpenStack\* Zun with Kata Containers](./use-cases/zun_kata.md): How to get Kata Containers to work with OpenStack Zun using DevStack on Ubuntu 16.04.
+* [SR-IOV with Kata Containers](./use-cases/using-SRIOV-and-kata.md): Setup to use SRIOV with Kata Containes and Docker.
```
Comment (GabyCT) [inline README.md:39]: s/Containes/Containers

File: README.md [hunk 2]
```diff
@@ -20,39 +20,39 @@ For details of the other Kata Containers repositories, see the
...
+* [Intel QAT with Kata Containers](./use-cases/using-Intel-QAT-and-kata.md): Setup instructions to download kernel source, compile kernel driver modules for source, load it, and prepare a specially-built Kata Containers kernel and rootfs.
```
Comment (GabyCT) [inline README.md:40]: `for source` / from source

File: README.md (hunk 1 above)
Comment (jodh-intel) [reply README.md:39]: Yep. Note that you can run the [spell checker](https://github.com/kata-containers/tests/tree/master/cmd/check-spelling) we used in the CI manually:

```sh
$ dir=~/go/src/github.com/kata-containers/
$ mkdir -p "$dir" && cd "$dir"
$ git clone https://github.com/kata-containers/tests
$ git clone https://github.com/kata-containers/documentation
$ cd documentation
$ ../tests/cmd/check-spelling/kata-spell-check.sh check ./README.md
```

File: README.md [hunk 3]
```diff
@@ -11,7 +11,7 @@
 * [Help Improving the Documents](#help-improving-the-documents)
 * [Website Changes](#website-changes)
 
-The [Kata Containers](https://github.com/kata-containers)
+The [Kata Containers\*](https://github.com/kata-containers)
```
Comment (grahamwhaley) [inline README.md:14]: Generally the Kata project has not used the `*` notation for 'copyright of others' in its docs. Probably best not to start adding them, unless you intend to add them to *all* the docs :-)

Comment (amshinde) [reply README.md:14]: Agree.

File: README.md [hunk 4]
```diff
@@ -20,39 +20,40 @@ For details of the other Kata Containers repositories, see the
...
 
-* [Installation guides](./install/README.md): Install and run Kata Containers with Docker or Kubernetes
+* [Installation guides](./install/README.md): 
+Install and run Kata Containers with Docker or Kubernetes
 
 ## More User Guides
 
-* [Upgrading](Upgrading.md): how to upgrade from [Clear Containers](https://github.com/clearcontainers) and [runV](https://github.com/hyperhq/runv) to [Kata Containers](https://github.com/kata-containers) and how to upgrade an existing Kata Containers system to the latest version.
-* [Limitations](Limitations.md): differences and limitations compared with the default [Docker](https://www.docker.com/) runtime,
+* [Upgrading](Upgrading.md): How to upgrade from [Intel Clear Containers](https://github.com/clearcontainers) and [runV](https://github.com/hyperhq/runv) to [Kata Containers](https://github.com/kata-containers) and how to upgrade an existing Kata Containers system to the latest version.
```
Comment (grahamwhaley) [inline README.md:28]: nb: we tend to keep our markdown lines trimmed to 80 chars or so - would you say @jodh-intel ?
I see it as a swings and roundabouts - it makes editing and reflowing lines sometimes more painful, and it has both pros and cons for reviewing patches of markdown (sometimes a win, sometimes a loss).

File: README.md [hunk 5]
```diff
@@ -20,39 +20,40 @@ For details of the other Kata Containers repositories, see the
...
 
-* [GPU Passthrough with Kata](./use-cases/GPU-passthrough-and-Kata.md)
-* [OpenStack Zun with Kata Containers](./use-cases/zun_kata.md)
-* [SR-IOV with Kata](./use-cases/using-SRIOV-and-kata.md)
-* [Intel QAT with Kata](./use-cases/using-Intel-QAT-and-kata.md)
-* [VPP with Kata](./use-cases/using-vpp-and-kata.md)
-* [SPDK vhost-user with Kata](./use-cases/using-SPDK-vhostuser-and-kata.md)
+* [GPU Passthrough with Kata Containers](./use-cases/GPU-passthrough-and-Kata.md): Kata Containers supports passing certain GPUs from the host into the container.
+* [OpenStack Zun with Kata Containers](./use-cases/zun_kata.md): How to get Kata Containers to work with OpenStack Zun using DevStack on Ubuntu 16.04.
+* [SR-IOV with Kata Containers](./use-cases/using-SRIOV-and-kata.md): Setup to use SRIOV with Kata Containers and Docker.
```
Comment (grahamwhaley) [inline README.md:40]: s/SRIOV/SR-IOV/ ?

File: README.md (hunk 4 above)
Comment (mvincerx) [reply README.md:28]: In my experience working on Clear Linux documentation, we set a standard 78 char limit, here: https://docs.01.org/clearlinux/latest/collaboration/structure-formatting.html#white-space-and-line-length.  80 chars seems fine too; I'm not commenting on that.  

Over time, this has proven more useful than not. Of course not all contributors follow this, but  our internal team does. Using this standard helps our team avoid common errors, like code-block syntax breaking due to improper line breaks, or Sphinx directives that are split over line breaks, which also fail. There's also a lot of research that shows humans are more likely to read, or read more, if the char limit is set to around 80.

File: README.md (hunk 5 above)
Comment (DougTW) [reply README.md]: Changed SRIOV to SR-IOV and Vector Package Processing to Vector Packet Processing.

**Comment by fidencio** [comment]
@DougTW, we're missing a `Signed-off-by` tag in at least one of your commits.  Please, add that and do a force push to this repo.

**Review by GabyCT — APPROVED** [review]

**Review by cmaf — APPROVED** [review]

**Comment by fidencio** [comment]
This PR, although would be good to have, doesn't contain anything critical.
I sincerely think we should evaluate this one as part of our [documentation efforts ](https://github.com/kata-containers/kata-containers/issues/1578).

As kata-containers 1.x will reach its EOL by May 12th, and we're working on getting everything either merged or closed, this will end up being closed.

Sorry for the time it was stale, and we promise we will try to avoid things like this happening as part of the https://github.com/kata-containers/kata-containers.

**Pull request closed without merging**